tests/longevity-tests/Dockerfile.6.5

```diff
@@ -15,4 +15,4 @@
 FROM longevity-base
 ENV GOVC_URL=blinky.eng.vmware.com
 ENV TEST_URL_ARRAY=blinky.eng.vmware.com
```

Comment:
Don't we want this off of blinky and onto some other testbed?

Reply:
yea, that needs to be changed in both 6.5 and 6.0 otherwise we are not using your infra at all @lgayatri   these URLs need to point to your 6.5 and 6.0 VC address